implementations/rust/ockam/ockam_command/src/help.rs

```diff
@@ -26,10 +18,9 @@ Feedback:
     on Github https://github.com/build-trust/ockam/discussions/new
 ";
 
-pub(crate) fn template(background: &str) -> &'static str {
-    let mut template: String = TEMPLATE_TOP.to_owned();
+pub(crate) fn template(body: &str) -> &'static str {
+    let mut template: String = body.clone().to_owned();
```

Comment:
```suggestion
    let mut template = body.to_string();
```

Reply:
@twittner 
I'll have a look on the GPG key setup for signing and add the suggested change.

Ho do you typically handle changes during/after review? Is force pushing to the PR branches ok, or is there a way to enable squashing during merge?

Reply:
@hargut force pushing to PR branches is okay. We can sign your commit during the merge. Once you tackle @twittner's suggested change, I'll squash sign and merge for you.

Reply:
Awesome, thank you. Will merge shortly.